src/phase-management/phase-rules.json

```diff
@@ -0,0 +1,392 @@
+        "conditions": {
+          "all": [
+            {
+              "fact": "isOpen",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isPastScheduledEndTime",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isClosed",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "registrantCount",
+              "operator": "greaterThanInclusive",
+              "value": 1
+            }
+          ]
+        },
+        "event": {
+          "type": "canClose"
+        }
+      }
+    ],
+    "Submission": [
+      {
+        "name": "Submission Close",
+        "conditions": {
+          "all": [
+            {
+              "fact": "isClosed",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "isOpen",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "any": [
+                {
+                  "fact": "isPastScheduledEndTime",
+                  "operator": "equal",
+                  "value": true
+                },
+                {
+                  "all": [
+                    {
+                      "fact": "nextPhase",
+                      "operator": "equal",
+                      "value": "Iterative Review"
+                    },
+                    {
+                      "fact": "hasActiveUnreviewedSubmissions",
+                      "operator": "equal",
+                      "value": true
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        "event": {
+          "type": "canClose"
+        }
+      }
+    ],
+    "Review": [
+      {
+        "name": "Review Close",
+        "conditions": {
+          "all": [
+            {
+              "fact": "isOpen",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isPastScheduledEndTime",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "isClosed",
+              "operator": "notEqual",
+              "value": true
+            },
+            {
+              "fact": "allSubmissionsReviewed",
+              "operator": "equal",
+              "value": true
+            }
+          ]
+        },
+        "event": {
+          "type": "canClose"
+        }
+      }
+    ],
+    "IterativeReview": [
+      {
+        "name": "Iterative Review Close",
+        "conditions": {
+          "all": [
+            {
+              "fact": "isOpen",
+              "operator": "equal",
+              "value": true
+            },
+            {
+              "fact": "hasActiveUnreviewedSubmissions",
```

Comment:
need a different fact to check if the earliest active submission has enough commited review(s)